experimental/IntersectionTheory/src/Bott.jl

```diff
@@ -80,10 +80,91 @@ The type of a abstract_variety with a torus action, represented by the fixed poi
   end
 end
 
-euler(X::TnVariety) = sum(1//ZZ(e) for (p,e) in X.points) # special case of Bott's formula
+@doc raw"""
+    tn_variety(n::Int, points::Vector{Pair{P, Int}}) where P
+
+Return an abstract_variety with a torus action, represented by the fixed points.
+"""
+tn_variety(n::Int, points::Vector{Pair{P, Int}}) where P = TnVariety(n::Int, points::Vector{Pair{P, Int}}) where P
+
+@doc raw"""
+     dim(X::TnVariety)
+
+Return the dimension of `X`.
+
+# Examples
+```jldoctest
+julia> G = tn_grassmannian(2, 5);
+
+julia> dim(G)
+6
+
+```
+"""
+dim(X::TnVariety) = X.dim
+
+@doc raw"""
+     fix_points(X::TnVariety)
+
+Return the fix points representing `X`.
```

Comment:
Please double check the spelling of "fix points" I believe it is 'fixed points'.